firetower~

```diff
@@ -0,0 +1,244 @@
+#!/bin/bash
```

Comment:
if your editor is creating temp files with a `~` suffix, you might want to add
```
*~
```
to your `.gitignore_global` file, and register it in your git config - (see https://help.github.com/articles/ignoring-files/#create-a-global-gitignore)